src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -259,7 +261,9 @@
     clearBtn.addEventListener('click', clearFilters, false);
     filterBtn.addEventListener('click', toggleFiltersVisibility, false);
     contentTypeSelect.addEventListener('click', toggleContentTypeSelectorVisibility, false);
-    sectionSelect.addEventListener('change', toggleDisabledStateOnApplyBtn, false);
+    if (sectionSelect) {
+        sectionSelect.addEventListener('change', toggleDisabledStateOnApplyBtn, false);
+    }
```

Comment:
Before and after `if` statement you should an empty line, for better readability.

Reply:
Please, apply this rule to all similar changes in your JS code.